docs/backtesting.md

```diff
@@ -33,22 +33,10 @@ freqtrade backtesting
 freqtrade backtesting --ticker-interval 1m
 ```
 
-#### Update cached pairs with the latest data
-
-```bash
-freqtrade backtesting --refresh-pairs-cached
-```
-
-#### With live data (do not alter your testdata files)
-
-```bash
-freqtrade backtesting --live
-```
-
 #### Using a different on-disk ticker-data source
 
```

Comment:
"Assume you downloaded the history data from the Bittrex exchange and kept it in the `user_data/data/bittrex-20180101` directory. You can then use this data for backtesting as follows:"
(pls correct wording here if needed)